DrivewayDentDeletion/Operators/pipeline-apic.yaml

```diff
@@ -36,7 +36,15 @@ spec:
         name: run-command
       params:
         - name: command
-          value: /workspace/git-source/DrivewayDentDeletion/Operators/deploy-im.sh -n {{NAMESPACE}} -b {{DEFAULT_BLOCK_STORAGE}} -f {{DEFAULT_FILE_STORAGE}}
+          value: |
+            FORKED_REPO={{FORKED_REPO}}
+            BRANCH={{BRANCH}}
+            API_FILE='["'${FORKED_REPO%.*}/raw/${BRANCH}/DrivewayDentDeletion/Bar_files/ace-api/DrivewayDemo.bar'"]'
+            ACME_FILE='["'${FORKED_REPO%.*}/raw/${BRANCH}/DrivewayDentDeletion/Bar_files/ace-acme/AcmeV1.bar'"]'
+            BERNIE_FILE='["'${FORKED_REPO%.*}/raw/${BRANCH}/DrivewayDentDeletion/Bar_files/ace-bernie/BernieV1.bar'"]'
+            CHRIS_FILE='["'${FORKED_REPO%.*}/raw/${BRANCH}/DrivewayDentDeletion/Bar_files/ace-chris/CrumpledV1.bar'"]'
+            CONFIGURATIONS="[keystore-ddd, policyproject-ddd-dev, serverconf-ddd, setdbparms-ddd, application-ddd-dev, barauth-empty]"
```

Comment:
Thinking about it maybe should pass FORKED_REPO and BRANCH to deploy-im.sh and get deploy-im.sh to do this step (setting of API_FILE, ACME_FILE, BERNIE_FILE, CHRIS_FILE and CONFIGURATIONS). That would avoid having to duplicate this for every pipeline, and also means the logic is in the script that is fetched at the start of the pipeline run, so if that is changed in git and committed/pushed the pipeline will trigger and pick it up. Stuff that is "baked" into the pipeline yaml doesn't get picked up automatically when the pipeline is triggered, the cicd scripts have to be re-run.

Reply:
I had thought about doing something where we pass in `"'${FORKED_REPO%.*}/raw/${BRANCH}/'"` to the script would that do the same as what you are suggesting?

Reply:
That sounds fine too, so passing in the git base url.

Just a note for configurations, they are slightly different for dev/test so you will need to pass in a parameter for which DDD env it is.

Reply:
got it 👍